### ISSUE TITLE
Can't activate account

### ISSUE THREAD

**Issue by ggommezz** [gpodder/mygpo#596]
Registered my new gpodder account on 5th June, got the activation email but the link provided did not work as it looked like gpodder was down or having problems at the time.  Is it possible to send me a new activation email please?

**Comment by ggommezz** [comment]
Note this account was for user name gomezz (not ggommezz which I created today so I could post this request).

**Comment by ggommezz** [comment]
Or alternatively do I now just delete the gomezz account and run with ggommezz?

**Comment by ryoung29** [comment]
duplicate

**Comment by ggommezz** [comment]
Huh?  Why close this topic without resolution of the issue?

**Comment by ryoung29** [comment]
Sorry @ggommezz the issue in general is resolved, but your issue remains. I'll work on it this week.

**Comment by lucker999** [comment]
+1
Can't register as well since no email came from gpodder.net

**Comment by mogwa1** [comment]
Same here: no email from gpodder.net is sent upon registering.
I can see that gpodder.net is connecting to my mail server, but it disconnects immediately without transfering a message.

**Comment by lucker999** [comment]
It appears to be some kind of a black list for certain public email services. For instance, when I registered with an email at mail.ru no email received (so I added a comment here). After a few days I registered with gmail.com and instantly received confirmation email.

**Comment by mogwa1** [comment]
Indeed, it was similar for me.  When I created another account with a gmail address, I got the registration email instantly.

I'm not sure it's blacklisting, though.  On my first attempt, I tested with an email address on an SMTP server that I administer myself.  I can see gpodder.net connect to SMTP, but it disconnects without sending a message.  That's before the message would hit any filter.

**Comment by lucker999** [comment]
@mogwa1 I meant a kind of blacklist on gpodder side. I suspect they just ignore some email services or allow only selected email services

**Comment by mewland** [comment]
Just registered with the Outlook mailbox. I got the activation email, but when I click on the link I see «The activation link is either not valid or has already expired».
